environment.yml

```diff
@@ -5,57 +5,66 @@ channels:
   - plotly
   - defaults
 dependencies:
+  # mdanalysis dependencies
   - cython
```

Comment:
It's odd that we only cover a small subset of the MDA setup and install requirements:

setup requirements: [Cython, packaging, numpy>=1.22.3, setuptools, wheel]
additional install requirements: [biopython>=1.8, networkx>=2.0, GridDataFormats>=0.4.0, mmtf-python>=1.0.0, joblib>=0.12, matplotlib>=1.5.1, tqdm>=4.43.0, threadpoolctl, fasterners]

Reply:
These might be covered by the `pip` install of `develop` MDA -- iirc only cython and numpy are crucial to have before installing MDA. At least CI is correctly building from this environment. I'd lean towards having as few dependencies here as possible so if core MDA updates its dependencies, it's reflected here as well.

Reply:
Even cython and numpy aren't crucial. They get picked up on wheel creation (the magic of our pyproject.toml). I'd advocate for getting rid of all the non optional dependencies then (although it does add to the risk of a dependency resolution failure).

Reply:
Personally I'd be happy with as minimal MDA requirements as possible here.

Reply:
Agreed but it'll take a lot of iteration to remove everything but what's absolutely needed